app/src/server/router/homes.ts

```diff
@@ -0,0 +1,97 @@
+import { createProtectedRouter } from "./context";
+import { z } from "zod";
+import { getHomesByUserId, canUserViewHome } from "../db/HomeService";
+
+export const homesRouter = createProtectedRouter()
```

Comment:
What is the difference between a protected router vs a normal router? Something that should be used on pages with authentication required?

Reply:
Protected router ensures that a user is signed in before executing any of the queries or mutations within the router. This is for ensuring that no unauthenticated users can tamper with the system